tests/TestHelpers/WebDavHelper.php

```diff
@@ -671,30 +673,44 @@ public static function getMtimeOfFileinPublicLinkShare(
 	 * @param string|null $baseUrl
 	 * @param string|null $resource
 	 * @param string|null $xRequestId
+	 * @param int|null $davPathVersionToUse
 	 *
 	 * @return string
-	 * @throws Exception
+	 * @throws GuzzleException
 	 */
 	public static function getMtimeOfResource(
 		?string $user,
 		?string $password,
 		?string $baseUrl,
 		?string $resource,
-		?string $xRequestId = ''
+		?string $xRequestId = '',
+		?int $davPathVersionToUse = 1
 	):string {
 		$response = self::propfind(
 			$baseUrl,
 			$user,
 			$password,
 			$resource,
 			["getlastmodified"],
-			$xRequestId
+			$xRequestId,
+			"0",
+			"files",
+			$davPathVersionToUse
 		);
 		$responseXmlObject = HttpRequestHelper::getResponseXml(
 			$response,
 			__METHOD__
 		);
 		$xmlpart = $responseXmlObject->xpath("//d:getlastmodified");
+		Assert::assertIsArray(
+			$xmlpart,
+			__METHOD__ . " last modified expression is not found in the XML. Expected 'xmlPart' to be an array but, found: " . $xmlpart
+		);
+		Assert::assertArrayHasKey(
+			0,
+			$xmlpart,
+			__METHOD__ . " XML part does not have key 0. Expected a value at index 0 of 'xmlPart' but, found: " . $xmlpart[0]
```

Comment:
```suggestion
			__METHOD__ . " XML part does not have key 0. Expected a value at index 0 of 'xmlPart' but, found: " . $xmlpart
```
the key does not exist if that assertion fires